From 17e2de6baed2c9eae391b040b5dd43ee2687aa95 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 23 May 2023 10:16:37 -0700
Subject: [PATCH] Avoid printing error summary

Autospec already detects the unpackaged file listing and other errors
that might appear at the end of a build, so there's no need to duplicate
the output.

Also, we occasionally see the summary output interleaved with the
original unpackaged file listing, so hopefully suppressing the summary
will avoid that issue.

Ported-by: William Douglas <william.douglas@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: William Douglas <william.douglas@intel.com>
---
 build/build.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/build.c b/build/build.c
index 632ef66..62f8ffe 100644
--- a/build/build.c
+++ b/build/build.c
@@ -460,12 +460,14 @@ exit:
 	    rpmlogPrintByMask(NULL, maskWarn);
 	}
 
+#if 0
 	if (rc != RPMRC_OK && rc != RPMRC_MISSINGBUILDREQUIRES) {
 	    if (rpmlogGetNrecsByMask(maskErrs)) {
 		rpmlog(RPMLOG_NOTICE, _("\nRPM build errors:\n"));
 		rpmlogPrintByMask(NULL, maskErrs);
 	    }
 	}
+#endif
     }
 
     rpmugFree();
-- 
2.40.1

